Lib/test/test_ssl.py

```diff
@@ -989,7 +989,7 @@ def test_load_verify_cadata(self):
 
 
     def test_load_dh_params(self):
-        filename = u'dhpäräm.pem'
+        filename = 'dhparam.pem'
```

Comment:
You misunderstood the purpose of the test: the test checks that non-ASCII characters are encoded properly.

Reply:
Forgive me, as I haven't read over everything extensively.  But it fails every time, resulting in that test being skipped.  Wouldn't it be better if lines 1003-1006 in commit 6e8f395001b026daea047cf225dcca5a973ae824 were removed altogether?  Or am I missing something bigger?